docs/data/hubble/developer-guide/backfill/JS-UDF.mdx

```diff
@@ -0,0 +1,150 @@
+---
+title: "Backfill using JS UDF"
+sidebar_position: 0
+---
+
+This document outlines methods to extract required fields from the XDR of raw data. We'll take the example of extracting the `fee_account_muxed` field from a transaction envelope (`tx_meta` XDR). However, this method can be adapted to other fields as well. It is worth noting that most users will not need to standup and run their own Hubble. The Stellar Development Foundation provides public access to the data through the public datasets and tables in GCP BigQuery. Instructions on how to access this data can be found in the [Connecting](../../developer-guide/connecting-to-bigquery/README.mdx) section.
```

Comment:
I think this is missing a line that gives context as to why you would need to do this. Something simple like Hubble does not parse every single field available in raw XDR, but it does save the raw transaction meta in case you need to extract a field directly from the XDR.

Reply:
Addressed in https://github.com/stellar/stellar-docs/pull/1362/commits/625b3bca8aa0fbd8a896a748558057e61ae863d3